apps/server/src/state.ts

```diff
@@ -0,0 +1,92 @@
+import { OntimeEvent, Playback, TimerState, TimerType } from 'ontime-types';
+import { clock } from './services/Clock.js';
+import { RestorePoint } from './services/RestoreService.js';
+import { calculateDuration } from 'ontime-utils';
+import { getCurrent } from './services/timerUtils.js';
+
+type TState = {
+  playback: Playback;
+  readonly timer: TimerState & {
+    readonly finishedNow: boolean;
+  };
+};
+
+export const state: TState = {
+  playback: Playback.Stop,
+  timer: {
+    clock: clock.timeNow(),
+    current: null,
+    elapsed: null,
+    expectedFinish: null,
+    addedTime: 0,
+    startedAt: null,
+    finishedAt: null,
+    secondaryTimer: null,
+    selectedEventId: null,
+    duration: null,
+    timerType: null,
+    endAction: null,
+    get finishedNow() {
+      return this.current <= 0 && this.finishedAt === null;
+    },
```

Comment:
I opted not to have getters for everything and to allow you to import `state` directly; it feels more straightforward. We can have a derived state using JavaScript getters.